docs/mesos/ManualInstallCalicoMesos.md

```diff
@@ -0,0 +1,186 @@
+<!--- master only -->
+> ![warning](../images/warning.png) This document applies to the HEAD of the calico-docker source tree.
+>
+> View the calico-docker documentation for the latest release [here](https://github.com/projectcalico/calico-docker/blob/v0.13.0/README.md).
+<!--- else
+> You are viewing the calico-docker documentation for release **release**.
+<!--- end of master only -->
+
+# Manually Install Calico and Mesos on Agent
+This tutorial will walk you through installing Mesos, Netmodules, and Calico onto a Centos host to create a Mesos agent compatible with Calico.
+
+## Prerequisites
+The sections below ensure that your Mesos agent is properly configured to allow Calico networking.
+
+### Set & verify fully qualified domain name
+These instructions assume each host can reach other hosts using their fully qualified domain names (FQDN).  To check the FQDN on a host use
+
+```
+hostname -f
+```
+Then attempt to ping that name from other hosts.
+
+It is also important that Calico and Mesos have the same view of the (non-fully-qualified) hostname.  Ensure that the value returned by `hostname` is unique for each host in your cluster.
+
+### Configure firewall
+You will either need to configure the firewalls on each node in your cluster (recommended) to allow access to the cluster services or disable them completely.  This section contains configuration examples for `firewalld`.  If you use a different firewall, check your documentation for how to open the listed ports.
+
+Agent (compute/slave) nodes require
+
+| Service Name | Port/protocol     |
+|--------------|-------------------|
+| BIRD (BGP)   | 179/tcp           |
+| mesos-agent  | 5051/tcp          |
+
+Example `firewalld` config:
+
+```
+sudo firewall-cmd --zone=public --add-port=179/tcp --permanent
+sudo firewall-cmd --zone=public --add-port=5051/tcp --permanent
+sudo systemctl restart firewalld
+```
+
+## 1. Download the Calico Mesos Plugin
+The Calico-Mesos plugin is available for download from the [calico-mesos repository releases](https://github.com/projectcalico/calico-mesos/releases). In this example, we will install the binary to the `/calico` directory.
+
+```
+wget https://github.com/projectcalico/calico-mesos/releases/download/v0.1.1/calico_mesos
+chmod +x calico_mesos
+sudo mkdir /calico
+sudo mv calico_mesos /calico/calico_mesos
+```
+## 2. Create the modules.json Configuration File
+To enable Calico networking in Mesos, you must create a `modules.json` file. When provided to the Mesos Agent process, this file will connect Mesos with the Net-Modules libraries as well as the Calico networking plugin, thus allowing Calico to 
+receive networking events from Mesos.
+
+```
+wget https://github.com/projectcalico/calico-containers/raw/master/docs/mesos/config/modules.json
```

Comment:
Can you change this to:
`wget https://raw.githubusercontent.com/projectcalico/calico-mesos/master/packages/sources/modules.json`

I want to minimize the copies of that `modules.json` file we're hosting. I know the one I linked to currently has the incorrect libmesos_network_isolator.so path, but its about to be fixed in this PR:
https://github.com/projectcalico/calico-mesos/pull/37/files